src/test/java/MockingTest.java

```diff
@@ -0,0 +1,33 @@
+import com.epam.tamentoring.bo.DiscountUtility;
+import com.epam.tamentoring.bo.OrderService;
+import com.epam.tamentoring.bo.UserAccount;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+class MockingTest extends AbstractTest {
+
+    @Mock
+    DiscountUtility discountUtility;
+
+    @InjectMocks
+    OrderService orderService = new OrderService();
+
+    @Test
+    void checkDiscountUtility() {
+        double returnedDiscount = 3.0;
+        UserAccount user = generateUserAccount();
+        double totalPriceInCartBefore = user.getShoppingCart().getCartTotalPrice();
+
+        Mockito.when(discountUtility.calculateDiscount(user)).thenReturn(returnedDiscount);
+        assertEquals(totalPriceInCartBefore - returnedDiscount, orderService.getOrderPrice(user));
+        Mockito.verify(discountUtility, Mockito.times(1)).calculateDiscount(user);
+        Mockito.verifyNoMoreInteractions(discountUtility);
```

Comment:
In my opinion, this validation no needed since there is only one method in discountUtility

Reply:
Ok, then you did everything right.